lib/globals.d.ts

```diff
@@ -21,3 +21,49 @@ declare module '*.json' {
   const value: Record<string, any>;
   export = value;
 }
+
+// these  declartions will be removed once the types.ts file for @renovate/pep440 is ready
+interface parsed {
```

Comment:
How about more specific name, something like `Pep440Version` or just `Pep440`?

Reply:
This one has been addressed in @renovatebot/pep440. We are using `Pep440Version`